src/Libraries/CoreNodes/Types.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace DSCore
 {
+    [SupressImportIntoVM]
     public class Types
```

Comment:
Can't this be made internal?? I'll prefer to make it internal rather than this attribute.